g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/expt/ExceptionHandle.java

```diff
@@ -28,6 +30,9 @@ public APIResponseDTO handler(Exception e) {
 			logger.error(e.getMessage(), e);
 			
 			response.setResponse(APIResponseStatus.INTERNAL_SERVER_ERROR);
+		} else if(e instanceof VIPAPIException){
```

Comment:
Use VIPAPIException in i18n-manager and use L10nAPIException in l10n-manager

Reply:
done